homeassistant/components/lock/wink.py

```diff
@@ -4,11 +4,54 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/lock.wink/
 """
+import logging
+from os import path
+
+import voluptuous as vol
+
 from homeassistant.components.lock import LockDevice
 from homeassistant.components.wink import WinkDevice, DOMAIN
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import ATTR_ENTITY_ID, STATE_UNKNOWN
+from homeassistant.config import load_yaml_config_file
 
 DEPENDENCIES = ['wink']
 
+_LOGGER = logging.getLogger(__name__)
+
+SERVICE_SET_VACATION_MODE = 'wink_set_lock_vacation_mode'
+SERVICE_SET_ALARM_MODE = 'wink_set_lock_alarm_mode'
+SERVICE_SET_ALARM_SENSITIVITY = 'wink_set_lock_alarm_sensitivity'
+SERVICE_SET_ALARM_STATE = 'wink_set_lock_alarm_state'
+SERVICE_SET_BEEPER_STATE = 'wink_set_lock_beeper_state'
+
+ATTR_ENABLED = 'enabled'
+ATTR_SENSITIVITY = 'sensitivity'
+ATTR_MODE = 'mode'
+
+ALARM_SENSITIVITY_MAP = {"low": 0.2, "medium_low": 0.4,
+                         "medium": 0.6, "medium_high": 0.8,
+                         "high": 1.0}
+
+ALARM_MODES_MAP = {"tamper": "tamper",
+                   "activity": "alert",
+                   "forced_entry": "forced_entry"}
+
+SET_ENABLED_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+    vol.Required(ATTR_ENABLED): cv.string,
+})
+
+SET_SENSITIVITY_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+    vol.Required(ATTR_SENSITIVITY): lambda x: ALARM_SENSITIVITY_MAP[x]
```

Comment:
Thanks, I tried to look for something like that but couldn't find anything. Easy to find once you know what it is called.